### PR TITLE
Graph application statuses as an image

### PR DESCRIPTION
**Pull request by benilovj**
### Context

We're prototyping the application statuses using the AASM gem.

### Changes proposed in this pull request

Add a way to generate an image of the state machine graph, via a rake task.

@alexdesi I think you had this in mind a couple of weeks ago

### CODE REVIEW

File: lib/tasks/create_application_status_graph.rake [hunk 1]
```diff
@@ -0,0 +1,5 @@
+task :create_application_status_graph => :environment do
+  application = CandidateApplication.create!
+  AASMDiagram::Diagram.new(application.aasm, 'tmp/application.png')
+  application.delete
+end
```
Comment (alexdesi) [inline lib/tasks/create_application_status_graph.rake]: Maybe we do not need to create and delete the application (`.new` should be enough):
```
  application = CandidateApplication.new
  AASMDiagram::Diagram.new(application.aasm, 'tmp/application.png')
```

**Comment by alexdesi** [comment]
Rubocop is not happy with `task :create_application_status_graph => :environment do`

File: lib/tasks/create_application_status_graph.rake (hunk 1 above)
Comment (benilovj) [reply lib/tasks/create_application_status_graph.rake]: yup - fixed

**Comment by benilovj** [comment]
Thanks @alexdesi I've fixed up both of the things you flagged

**Pull request merged**